src/include/binder/binder.h

```diff
@@ -3,6 +3,7 @@
 #include "binder/query/bound_regular_query.h"
 #include "common/copier_config/copier_config.h"
 #include "expression_binder.h"
+#include "main/client_context.h"
```

Comment:
I would consider fwd declare this. It's better to not include main header in other module